.github/ISSUE_TEMPLATE/ui-copy-request.md

```diff
@@ -0,0 +1,38 @@
+---
+name: UI Copy Request
+about: Describe the UI text that you need us to work on.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Description 
+[Summarize your request here. What is the work, what is its current status, and how can we help? Feel free to include screenshots.]
+
+### Software version
+Does this work affect serverless, stateful/classic, or both? Is this work targeted for a particular release version? Is this work related to text that's already in production?
+
+### Timeline
+What is the timeline for this work? When would it be ideal for for us to complete the request? What deliverables do you need, and when? If applicable, how do the release timelines for this feature differ between serverless / stateful?
+
+### Collaborators
+Please include contact information for the responsible product manager, designer, and developer (name, github / slack @):
+
+PM:
+Designer: 
+Developer:
+
+## Links / related assets  
+
+Please include each of the following, if applicable:
+
+Figma link(s):
+
+Github epic link(s):
+
+Github issue link(s):
+
+Production page(s) where content appears (link, or describe how to find it):
```

Comment:
A few things:
- Tiny li'l nit, the parentheses aren't needed and feel overly formal. Folks will understand that they can submit one or more of each.
- Do we need to break out epics vs. issues? And we'd also want PR links if they have them, so maybe just one item for All The GitHub Things.
- In this context, maybe "production pages" means something like "areas of UI"?

```suggestion
Figma links:

GitHub links (epics, issues, PRs):

Areas of UI where content appears (link, or describe how to find it):
```

Also agree with @nastasha-solomon, we can remind folks not to share internal deployment credentials, though I didn't read this as requesting deployment info, just general info about where to find the text within the app. We need both, though, so wherever we ask about deployments should also have a reminder about internal stuff.